package.json

```diff
@@ -4,9 +4,9 @@
   "scripts": {
     "type-check": "tsc",
     "generate": "npm-run-all generate:*",
-    "generate:payment-ecommerce-v1": "rimraf generated/definitions/payment-ecommerce && shx mkdir -p generated/definitions/payment-ecommerce && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.323.0/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce --request-types --response-decoders --client",
-    "generate:payment-ecommerce-v2": "rimraf generated/definitions/payment-ecommerce-v2 && shx mkdir -p generated/definitions/payment-ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.323.0/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-v2 --request-types --response-decoders --client",
-    "generate:checkout-feature-flags": "rimraf generated/definitions/checkout-feature-flags && shx mkdir -p generated/definitions/checkout-feature-flags && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/refs/heads/3713-added-feature-flags/src/domains/checkout-app/api/checkout/checkout_feature_flags/v1/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/checkout-feature-flags --request-types --response-decoders --client",
+    "generate:payment-ecommerce-v1": "rimraf generated/definitions/payment-ecommerce && shx mkdir -p generated/definitions/payment-ecommerce && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.392.0/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce --request-types --response-decoders --client",
+    "generate:payment-ecommerce-v2": "rimraf generated/definitions/payment-ecommerce-v2 && shx mkdir -p generated/definitions/payment-ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.392.0/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-v2 --request-types --response-decoders --client",
+    "generate:checkout-auth-service-v1": "rimraf generated/definitions/checkout-auth-service-v1 && shx mkdir -p generated/definitions/checkout-auth-service-v1 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/62aaa8ca868d9c83aea8516b791dddc380dc8fe3/src/domains/checkout-app/api/checkout/checkout_auth_service/v1/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/checkout-auth-service-v1 --request-types --response-decoders --client",
```

Comment:
why replacing feature flags code generation here?

Reply:
It is present in line 10. Rebasing branch moved it